### PR TITLE
Docs - Add usage for data diagnosis

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
Add usage for data diagnosis.

### CODE REVIEW

File: docs/user-tutorial/data-diagnosis.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ id: data-diagnosis
...
+1. Install Superbench as [Superbench installation](https://microsoft.github.io/superbenchmark/docs/getting-started/installation) on the local machine.
+
+2. Prepare the raw data, rule file, baseline file under current path or somewhere on the local machine.
+
+3. After installing the superbnech and the files are ready, you can start to filter the defective machines automatically using  `sb result diagnosis` command.
+
+  ```
+  sb result diagnosis --data-file '${output_dir}/results-summary.jsonl' --rule-file './rule.yaml' --baseline-file './baseline.json' --output-file-foramt 'excel'/'json' --output-dir ${output-dir}
+  ```
+
```
Comment (cp5555) [inline docs/user-tutorial/data-diagnosis.md:22]: The detailed command can be found from xxx (reference cli.md)

File: docs/user-tutorial/data-diagnosis.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ id: data-diagnosis
 
 This tool is to filter the defective machines automatically from thousands of benchmarking results according to rules defined in **rule file**.
 
+## Usage
+
+1. Install Superbench as [Superbench installation](https://microsoft.github.io/superbenchmark/docs/getting-started/installation) on the local machine.
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/user-tutorial/data-diagnosis.md]: ok, thanks

File: docs/user-tutorial/data-diagnosis.md [hunk 3]
```diff
@@ -8,6 +8,20 @@ id: data-diagnosis
...
+
+2. Prepare the raw data, rule file, baseline file under current path or somewhere on the local machine.
+
+3. After installing the superbnech and the files are ready, you can start to filter the defective machines automatically using  `sb result diagnosis` command.
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/user-tutorial/data-diagnosis.md]: done

File: docs/cli.md [hunk 4]
```diff
@@ -172,3 +172,48 @@ Print version:
...
+
+| Name          | Default | Description        |
+|---------------|---------|--------------------|
+| `--help` `-h` | N/A     | Show help message. |
+
+#### Examples
+
+run data diagnosis and output the results in excel format:
+```bash title="SB CLI"
+sb result diagnosis --data-file 'outputs/results-summary.jsonl' --rule-file 'rule.yaml' --baseline-file 'baseline.json' --output-file-foramt 'excel'
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/cli.md]: done

File: docs/cli.md [hunk 5]
```diff
@@ -172,3 +172,48 @@ Print version:
...
+| `--baseline-file` `-b` | Path to baseline file. |
+| `--data-file` `-d`     | Path to raw data file. |
+| `--rule-file` `-r`     | Path to rule file.     |
+
+#### Optional arguments
+
+| Name                   | Default | Description                                                                 |
+|------------------------|---------|-----------------------------------------------------------------------------|
+| `--output-dir`         | `None`  | Path to output directory, outputs/{datetime} will be used if not specified. |
+| `--output-file-format` | `excel` | Format of output file, excel or json.  Default: excel.                      |
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/cli.md]: done

File: docs/cli.md [hunk 6]
```diff
@@ -172,3 +172,48 @@ Print version:
 ```bash title="SB CLI"
 sb version
 ```
+
+### `sb result diagnosis`
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/cli.md]: ok, thanks

File: docs/user-tutorial/data-diagnosis.md [hunk 7]
```diff
@@ -8,6 +8,20 @@ id: data-diagnosis
...
+
+  ```
+  sb result diagnosis --data-file '${output_dir}/results-summary.jsonl' --rule-file './rule.yaml' --baseline-file './baseline.json' --output-file-foramt 'excel'/'json' --output-dir ${output-dir}
```
[the remark this answers is not in the document]
Comment (yukirora) [reply docs/user-tutorial/data-diagnosis.md]: done

**Review by cp5555 — APPROVED** [review]

File: docs/user-tutorial/data-diagnosis.md [hunk 8]
```diff
@@ -8,6 +8,20 @@ id: data-diagnosis
 
 This tool is to filter the defective machines automatically from thousands of benchmarking results according to rules defined in **rule file**.
 
+## Usage
+
+1. [Install SuperBench](./getting-started/installation) on the local machine.
+
+2. Prepare the raw data, rule file, baseline file under current path or somewhere on the local machine.
+
+3. After installing the Superbnech and the files are ready, you can start to filter the defective machines automatically using  `sb result diagnosis` command. The detailed command can be found from [Superbench cli](https://microsoft.github.io/superbenchmark/docs/cli).
```
Comment (abuccts) [inline docs/user-tutorial/data-diagnosis.md:17]:
```suggestion
3. After installing the Superbnech and the files are ready, you can start to filter the defective machines automatically using  `sb result diagnosis` command. The detailed command can be found from [SuperBench CLI](../cli).
```

File: docs/cli.md [hunk 9]
```diff
@@ -101,6 +101,51 @@ Execute GPT2 model benchmark in default configuration:
...
+
+#### Global arguments
+
+| Name          | Default | Description        |
+|---------------|---------|--------------------|
+| `--help` `-h` | N/A     | Show help message. |
+
+#### Examples
+
+run data diagnosis and output the results in excel format:
```
Comment (abuccts) [inline docs/cli.md:139]: run -> Run

File: docs/cli.md [hunk 10]
```diff
@@ -101,6 +101,51 @@ Execute GPT2 model benchmark in default configuration:
...
+```bash title="SB CLI"
+sb result diagnosis --data-file outputs/results-summary.jsonl --rule-file rule.yaml --baseline-file baseline.json --output-file-foramt excel
+```
+
+run data diagnosis and output the results in jsonl format:
```
Comment (abuccts) [inline docs/cli.md:144]: same

**Review by abuccts — APPROVED** [review]

File: docs/user-tutorial/data-diagnosis.md (hunk 8 above)
Comment (yukirora) [reply docs/user-tutorial/data-diagnosis.md]: updated

File: docs/cli.md (hunk 9 above)
Comment (yukirora) [reply docs/cli.md]: upated

**Pull request merged**